web/src/components/Dialogs/ImageDialog.tsx

```diff
@@ -56,25 +64,41 @@ export default function ImageDialog({
 
   //#endregion
 
+  //#region Derived
+
+  const downloadLink = src.replace("_compressed", "");
```

Comment:
So faiz set up his python backend to upload two images: one in the original size (without _compressed) and another one compressed for fetching. So this essentially removes that compressed part of the url to fetch the original one. This doesn't apply to any of the newer images since we are now using replicate

Reply:
I see, makes sense